spec/requests/user_books_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.describe "UserBooks", type: :request do
+RSpec.describe Api::UserBooksController, type: :request do
```

Comment:
FYI I held off commenting too much in the tests. Will add that when you have a PR addressing your tests.